pkgs/development/python-modules/humanize/default.nix

```diff
@@ -22,7 +22,7 @@ buildPythonPackage rec {
     owner = "python-humanize";
     repo = pname;
```

Comment:
`repo` shouldn't be determined by `pname`.

```suggestion
    repo = "humanize";
```

Reply:
Done.